gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageGrpcWriteChannel.java

```diff
@@ -90,8 +93,12 @@
   private final String requesterPaysProject;
   private final BackOffFactory backOffFactory;
   private final Watchdog watchdog;
+  private final int MAX_BYTES_PER_MESSAGE = MAX_WRITE_CHUNK_BYTES.getNumber();
 
   private GoogleCloudStorageItemInfo completedItemInfo = null;
+  private LinkedBlockingQueue<ByteString> writeBuffer;
```

Comment:
Please add a comment explaining WHY we need these instance fields

Reply:
done